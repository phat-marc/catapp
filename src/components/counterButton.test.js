import { shallow } from 'enzyme';
import React from 'react';
import CounterButton from './counterButton';
// import renderer from 'react-test-renderer';

describe('counterButton', () => {
	it('expect to render counterButton component', () => {
		const mockColor = 'red'
		expect(shallow(<CounterButton color={mockColor}/>)).toMatchSnapshot();
	})

	// it('expect to render counterButton component', () => {
	// 	const mockColor = 'red'
	// 	const tree = renderer
	// 		.create(<CounterButton color={mockColor}/>)
	// 		.toJSON();
	// 	expect(tree).toMatchSnapshot();
	// })

	it('correctly increments the counter', () => {
		const mockColor = 'red'
		const wrapper = shallow(<CounterButton color={mockColor}/>);
			
		wrapper.find('[id="counter"]').simulate('click');
		wrapper.find('[id="counter"]').simulate('click');
		expect(wrapper.state()).toEqual({ count: 2 });
	})
})