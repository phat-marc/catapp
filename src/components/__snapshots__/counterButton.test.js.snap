// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`counterButton expect to render counterButton component 1`] = `ShallowWrapper {}`;
